Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -809,6 +809,12 @@ theorem mem_sep {p : ZFSet.{u} → Prop} {x y : ZFSet.{u}} :
           rwa [mk_func, ← ZFSet.sound h]⟩,
         h⟩⟩
 
+@[simp]
+theorem sep_empty f : (∅ : ZFSet).sep f = ∅ := by
```

Comment:
Oh I'm just noticing this, but can you provide the type of `f` explicitly? The syntax looks pretty weird otherwise.

Reply:
Sure, won't be able to for next few days. Will do this once I am able.

Reply:
I can push this change myself if you don't mind.

Reply:
Sure, thanks for all the help!